### PR TITLE
Bring back get kubeconfig-path

### PR DESCRIPTION
**Pull request by BenTheElder**
xref: https://github.com/kubernetes-sigs/kind/issues/850 
references: https://github.com/kubernetes-sigs/kind/issues/1060

depends on: https://github.com/kubernetes-sigs/kind/pull/1059

### CODE REVIEW

**Comment by BenTheElder** [comment]
```
$ kind get kubeconfig-path
`kind get kubeconfig-path` is deprecated!

KIND will export and merge kubeconfig like kops, minikube, etc.
This command is now unnecessary and will be removed in a future release.

For more info see: https://github.com/kubernetes-sigs/kind/issues/1060
See also the output of `kind create cluster`

/Users/bentheelder/.kube/config
```
(note that only the last line is stdout)


The only problem is that users that are doing `$(kind get kubeconfig-path)` will never see this...

Hopefully they check the release notes, which will include this information as well.

**Comment by aojea** [comment]
maybe we can remove it in 0.7.0 and warn it now, just using 0.6.0 as a deprecation cycle
This will give at least some notice period, what do you think?

**Comment by BenTheElder** [comment]
Right. That's what the comments say roughly I think?
This needs to exist when we release 0.6.0 but it can be removed at some point after.

File: pkg/cmd/kind/get/kubeconfigpath/kubeconfigpath.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+		"the cluster context name",
+	)
+	return cmd
+}
+
+func runE(logger log.Logger, streams cmd.IOStreams) error {
+	logger.Warn("`kind get kubeconfig-path` is deprecated!")
+	logger.Warn("")
+	logger.Warn("KIND will export and merge kubeconfig like kops, minikube, etc.")
+	logger.Warn("This command is now unnecessary and will be removed in a future release.")
```
Comment (aojea) [inline pkg/cmd/kind/get/kubeconfigpath/kubeconfigpath.go]: I guess we can replace `in a future release`  by `next release`?

Comment (BenTheElder) [reply pkg/cmd/kind/get/kubeconfigpath/kubeconfigpath.go]: that's confusing for anyone that is using it at head?
this warning doesn't need to be that specific, we will put it in the release notes too.
we can also choose not to remove it in 0.7 for some reason

Comment (tao12345666333) [reply pkg/cmd/kind/get/kubeconfigpath/kubeconfigpath.go]: > that's confusing for anyone that is using it at head?
> this warning doesn't need to be that specific, we will put it in the release notes too.
> we can also choose not to remove it in 0.7 for some reason

+1 .

**Comment by BenTheElder** [comment]
definitely underestimated https://github.com/search?p=11&q=%22kind+get+kubeconfig-path%22&type=Code before 🙃

**Comment by dims** [comment]
LOL indeed :)

**Pull request merged**